src/main/java/org/cbioportal/legacy/web/error/GlobalExceptionHandler.java

```diff
@@ -40,7 +40,7 @@
 // TODO
 // - consider extending extends ResponseEntityExceptionHandler
 // - check controllers for not catching exceptions themselves
-@ControllerAdvice("org.cbioportal.legacy.web")
+@ControllerAdvice({"org.cbioportal.legacy.web", "org.cbioportal.application.rest.vcolumnstore"})
```

Comment:
@haynescd seems like this just brings columnstore into line with legacy (re excewption handling).  seems reasonable?

Reply:
I'll defer to @haynescd on this, but maybe duplicating is worse than leaving a dependency which should be easy to detect and fix if and when we finally get rid of legacy codebase?